src/contexts/DanteFinance/DanteFinanceProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { createContext, useEffect, useState } from 'react';
+import { useWallet } from 'use-wallet';
+
+import config from '../../config';
+import DanteFinance from '../../dante-finance';
+
+export interface DanteFinanceContext {
+  danteFinance?: DanteFinance;
+}
+
+export const Context = createContext<DanteFinanceContext>({
```

Comment:
Context is too generic... and il also exported. Give a more significant name. E.g. DanteFinanceContext